spec/system/admin_customize_form_templates_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
```

Comment:
A high level note on specs here but I would recommend to organize things like this:

```
setup

action

expectation
```

It makes for a much easier to parse spec.